src/data/bucket.js

```diff
@@ -98,3 +98,9 @@ module.exports = {
         return output;
     }
 };
+
+export default exported;
+
+export const {
+    deserialize
+} = exported;
```

Comment:
We should probably get rid of all instances of exporting object literals like this — I suppose this is a legacy require/codemod thing? Ideally we would always export symbols directly, and only export default for classes/functions. This makes tree-shaking more effective.

Reply:
👍 yeah these cases were either codemod outputs (or just the quickest manual fix for the time being).  I'm not sure how much tree-shaking we'll enjoy from within our codebase, since I don't think we have too much dead code within `src/`... but if we do end up going the code-splitting route, this could make a notable difference in the worker blob size.   It'll also just be simpler/clearer code.

Reply:
Can you ticket this as a followup item?

Reply:
Yep, already did: https://github.com/mapbox/mapbox-gl-js/issues/6293